spec/ae_fast_decimal_formatter_spec.rb

```diff
@@ -0,0 +1,56 @@
+require 'spec_helper'
+
+describe 'AeFastDecimalFormatter' do
+  def ae_fast_decimal_formatter(number, precision)
+    AeFastDecimalFormatter.new(number, precision).format
```

Comment:
Is it possible to make the interface a class method?
Like `AeFastDecimalFormatter.format(number, precision)` to be consistent with what we currently have and it feels redundant to first initialize an object and call `format`.

Reply:
One thing I remember from Sandi Metz class is that class method is less flexible to extend than instance method.